bridges/bin/runtime-common/src/messages_generation.rs

```diff
@@ -104,6 +105,38 @@ where
 	(root, storage_proof)
 }
 
+/// Prepare storage proof of given messages delivery.
```

Comment:
This PR is the first PR, where this code is changed => no review for bridges changes yet. But I've just moved code around